### PR TITLE
Release notes 5.2.1

### PR DESCRIPTION
**Pull request by Chr1st0ph3rTurn3r**

### CODE REVIEW

File: docs/release_notes_128t_5.2.md [hunk 1]
```diff
@@ -10,6 +10,40 @@ If you do not see an issue listed below, it may have been resolved in another re
...
+## Release 5.2.1
+**Release Date:** July 9, 2021
+
+Contains fixes from: [Release 5.1.4, June 28, 2021](release_notes_128t_5.1.md#release-514)
+
+###  New Features and Improvements
+
+- **I95-35414 Refresh actions now available for individual sections on the Router Page:**The Device interface, Network Interface, and Peer Paths table sections now can be refreshed independently.
+------
+- **I95-36244 Handle names in application-id JSON:** The application-module json output common-name object now includes a list of referenced common-names, in addition to the transport-information list.
```
Comment (MichaelBaj) [inline docs/release_notes_128t_5.2.md:22]: I95-36244 is actually the defect: error display generated config in the PCLI. Can you correct the issue ID? Also, I'm not sure the description reads correctly.

File: docs/release_notes_128t_5.2.md [hunk 2]
```diff
@@ -10,6 +10,40 @@ If you do not see an issue listed below, it may have been resolved in another re
...
+------
+- **I95-38445 GUI Session Capture:** Added pages to the user interface that allow you to view and configure capture information.
+------
+- **I95-40458 Added the ability to toggle between Advanced and Basic Configuration mode:** Added the option to limit the main configuration screen to the most frequently used fields, or display all configuration options.
+
+### Resolved Issues
+
+- **I95-19871 Unknown session-type mismatched:** When a session-type does not have a match, the `Unclassified` service-class is used when it is available.**
+------
+- **I95-40075/I95-40134 Use gateway from service route for interface ping gateway:** the `service-route next-hop gateway-ip` will be used for ICMP ping reachability probe if so configured. If no IP is configured, then the `fib-route gateway` will be used, and finally the `network-interface gateway` will be used.
```
Comment (MichaelBaj) [inline docs/release_notes_128t_5.2.md:34]: Capitalize "The"

**Review by MichaelBaj — CHANGES_REQUESTED** [review]

File: docs/release_notes_128t_5.2.md [hunk 3]
```diff
@@ -10,6 +10,40 @@ If you do not see an issue listed below, it may have been resolved in another re
...
 :::
 
+## Release 5.2.1
+**Release Date:** July 9, 2021
+
+Contains fixes from: [Release 5.1.4, June 28, 2021](release_notes_128t_5.1.md#release-514)
+
+###  New Features and Improvements
+
+- **I95-35414 Refresh actions now available for individual sections on the Router Page:**The Device interface, Network Interface, and Peer Paths table sections now can be refreshed independently.
```
[the remark this answers is not in the document]
Comment (Chr1st0ph3rTurn3r) [reply docs/release_notes_128t_5.2.md:20]: added space

File: docs/release_notes_128t_5.2.md (hunk 1 above)
Comment (Chr1st0ph3rTurn3r) [reply docs/release_notes_128t_5.2.md:22]: corrected the number (36224) and edited the description.

File: docs/release_notes_128t_5.2.md (hunk 2 above)
Comment (Chr1st0ph3rTurn3r) [reply docs/release_notes_128t_5.2.md:34]: done

**Review by MichaelBaj — APPROVED** [review]

**Review by MichaelBaj — APPROVED** [review]

**Review by MichaelBaj — APPROVED** [review]

**Review by jeffmart — APPROVED** [review]

**Pull request merged**